README.md

```diff
@@ -29,7 +29,7 @@ If you have any questions, queries or comments, [RocketChat] is the place to fin
 
 ## Besu Users
 
-Users wanting to install Besu should install the Besu binary by following [these instructions](https://besu.hyperledger.org/HowTo/Get-Started/Install-Binaries/).    
+Users want to install Besu should install the Besu binary by following [these instructions](https://besu.hyperledger.org/HowTo/Get-Started/Install-Binaries/).    
```

Comment:
Should it be "who want" instead?  @MadelineMurray

Reply:
Maybe it should just be "To install the Besu binary, follow these instruction ..."

Reply:
+1 to @macfarla's suggestion